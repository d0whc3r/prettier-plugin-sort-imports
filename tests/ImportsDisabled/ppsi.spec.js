run_spec(__dirname, ['typescript'], {
    importOrderDisabled: true
});
