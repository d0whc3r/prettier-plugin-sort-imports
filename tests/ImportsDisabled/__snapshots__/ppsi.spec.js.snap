// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`imports-without-change.ts - typescript-verify: imports-without-change.ts 1`] = `
import sample from 'sample';
import Asample from 'a-sample';
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
import sample from "sample";
import Asample from "a-sample";

`;
